app/js/components/encounters/encounterForm.jsx

```diff
@@ -119,7 +100,12 @@ const Encounter = (props) => {
             >
               {
                 locationArray.map(locations => (
-                  <option key={locations.uuid} value={locations.uuid}>{locations.display}</option>
+                  <option
+                    key={locations.uuid}
+                    value={locations.uuid}
+                  >
+                    {locations.display}
```

Comment:
Shouldn't you use name instead of display?

Reply:
@dkayiwa  I did consider that but noted when I make api calls without including v=full query, the json result included only display and that's why i opted to use it instead.